nix/keys.nix

```diff
@@ -133,6 +155,8 @@ in
               let
                 # FIXME: The key file should be marked as private once
                 # https://github.com/NixOS/nix/issues/8 is fixed.
+                #
+                # FIXME: implement value.filename for this as well
```

Comment:
I suggest we fail here if `text` is not set.

Given the discussion in https://github.com/NixOS/nixops/pull/650, this will change soon so no need to implement `filename` handling sanely.